articles/azure-monitor/app/managed-workspaces.md

```diff
@@ -62,18 +62,19 @@ Beginning in April 2025, classic Application Insights resources are automaticall
 - The workspace is placed in a new resource group. This new group doesn't inherit access permissions from the Application Insights resource group. However, users with appropriate permissions can still query telemetry data through the Application Insights resource, due to resource-centric access control.
 
 > [!IMPORTANT]  
-> Each migrated classic resource receives its own managed workspace. To prevent this scenario, [migrate your classic resources manually](/previous-versions/azure/azure-monitor/app/convert-classic-resource).
+> Each migrated classic resource receives its own managed workspace and resource group.  Due to an Azure limit on number of resource groups allowed in a subscription, the auto-migration process may cause your subscription to reach or come close to that limit and prevent additional resoruce groups from being created. To prevent this scenario, [migrate your classic resources manually](/previous-versions/azure/azure-monitor/app/convert-classic-resource).
```

Comment:
```suggestion
> Each migrated classic resource receives its own managed workspace and resource group.  Due to an Azure limit on the number of resource groups allowed in a subscription, the auto-migration process may cause your subscription to reach or come close to that limit and prevent additional resource groups from being created. To prevent this scenario, [migrate your classic resources manually](/previous-versions/azure/azure-monitor/app/convert-classic-resource).
```
fixing grammar and spelling